package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "discourse-tag-by-group",
+  "version": "0.1.0",
+  "repository": "https://github.com/discourse/discourse-tag-by-group",
+  "author": "Discourse",
+  "license": "UNLICENSED",
```

Comment:
```suggestion
  "license": "MIT",
```

Reply:
And we need a license file like [here](https://github.com/discourse/discourse-calendar/blob/main/LICENSE), I think.